pirant/urlbuilder.py

```diff
@@ -48,6 +48,7 @@ def get_collabs_url(self, skip, limit):
         skip = self.validate_int_input(skip)
         return str(self.collabs_url % (self.base_url, self.app_version, skip, limit))
 
+    @staticmethod
```

Comment:
This right here seems odd. 
Python, by default, pases `self` to class methods that are being called on an instance. I've removed this again in commit 62f4f2372766cafb2c6c5265ae0ff5ad4dbda8ca - the tests passed, but check for yourself if this is ok to do - perhaps I oversaw something that fall between the cracks of the test suite.